app/store/message.go

```diff
@@ -138,3 +131,31 @@ func (s *Session) GetMessages(i int) *Message {
 func (m *Message) GetName() string {
 	return TelToName(m.Source)
 }
+
+// FindQuotedMessage searches the equivalent message of DataMessage_Quote in our
+// DB and returns the local message id
+func FindQuotedMessage(quote *signalservice.DataMessage_Quote) (error, int64) {
+	var quotedMessages = []Message{}
+	err := DS.Dbx.Select(&quotedMessages, "SELECT * FROM messages WHERE sentat = ?", quote.GetId())
+	if err != nil {
+		return err, -1
+	}
+	if len(quotedMessages) == 0 {
+		return err, -1
+	}
+	id := quotedMessages[0].ID
+	return nil, id
+}
+
+// returns a message by it's ID
+func GetMessageById(id int64) (error, *Message) {
+	var quotedMessages = []Message{}
```

Comment:
I think this variable should be named `message` not `quotedMessage`.

Reply:
Good point :)